students/vvinodh/Lesson2/fibo_series.py

```diff
@@ -0,0 +1,14 @@
+def fibo(n):
+    #initial assignment based on where the series need to start
+    a = 0
+    b = 1
+    # If the required index is 0th position print the first fibonacci number
+    if n == 0:
+        print (a)
+    else:
+    # Loop to move the latest number to prior and new number to the next calculated variable
```

Comment:
you want to align your comment with the code block (line 10), so this should be indented by 4 spaces